raster-foundry/raster-foundry
# Fixup authorization in team and organization routes

## Overview

This PR:

- rips out all of the team routes accept the detail route, since they weren't used anywhere
- adds the `viewFilter` querybuilder... builder to list organizations in organization routes

### Checklist

- [X] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated (raster-foundry/raster-foundry-api-spec#35)

## Testing Instructions

 * Log in as an admin user
 * Wander around the admin section to list, get detail for, and update a team
 * Confirm all requests still succeed even though the routes are gone

Closes azavea/raster-foundry-platform#368
